### PR TITLE
Maya

### PR DESCRIPTION
**Pull request by Mahliyo27**

### CODE REVIEW

**Comment by Hicham0120** [comment]
.great job

File: src/utils/tests.java [hunk 1]
```diff
@@ -0,0 +1,53 @@
...
+            result +=name.charAt(i);
+        }
+
+        System.out.println(result);
+
+
+
+
+
+
```
Comment (Hicham0120) [inline src/utils/tests.java:32]: Looks good

**Review by Hicham0120 — APPROVED** [review]

**Comment by latifetor** [comment]
look good, keep up

**Comment by Mahliyo27** [comment]
Thanks

**Pull request merged**